src/components/ColorPicker/ColorPicker.scss

```diff
@@ -0,0 +1,364 @@
+@import "../../constants/style";
+
+.color-picker {
+  z-index: 220;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding-bottom: $spacing--xxs;
+  box-shadow: 0 0 20px rgba(55, 62, 79, 0.12);
```

Comment:
hard-coded, this color probably exists in the color definitions. do the following:
1. find out hex code of color by converting from rgb
2. find the color in `colors.scss` that has this value
3. use `rgba($the-right-color, 0.12)`